nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/test/java/org/apache/nifi/processors/standard/TestListenHTTP.java

```diff
@@ -298,6 +299,56 @@ public void testSecureTwoWaySslPOSTRequestsReceivedWithoutEL() throws Exception
         testPOSTRequestsReceived(HttpServletResponse.SC_OK, true, true);
     }
 
+    @Test
+    public void testSecureTwoWaySslPOSTRequestsReceivedWithAuthorizedSubjectDn() throws Exception {
```

Comment:
Is this test method necessary in light of existing unit tests that rely on the default Authorized DN Pattern?

Reply:
Good point, will remove this one.